### PR TITLE
Issue169 contribution agreement

### PR DESCRIPTION
**Pull request by mortenwh**
#169

### CODE REVIEW

File: source/definition_of_done.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@ Definition of Done
 The following requirements must be satisfied before a task can be considered done:
 
 * The solution is in agreement with the design
+* Software repositories are licensed (see :ref:`general-conventions`)
+* The headers of all software code files contain a reference to the repository license
```
[the remark this answers is not in the document]
Comment (mortenwh) [reply source/definition_of_done.rst]: @mpejcoch - I added a template text. Can you have a look?

**Comment by mortenwh** [comment]
@mpejcoch kan du sjekke denne og lukke den hvis det er ok?

File: source/definition_of_done.rst [hunk 2]
```diff
@@ -4,6 +4,8 @@ Definition of Done
 The following requirements must be satisfied before a task can be considered done:
 
 * The solution is in agreement with the design
+* Software repositories are licensed (see :ref:`general-conventions`)
+* The headers of all software code files contain a reference to the repository license (i.e., with a statement such as "License: This file is part of <repository>. <Repository> is licensed under <license name> (<link to license>)")
```
Comment (mpejcoch) [inline source/definition_of_done.rst:8]: Can you be even more concrete about how to do this, e.g.,
`<repository>` is

- [ ] S-ENDA prototype
- [ ] S-ENDA prototype hosted at github.com/metno
- [ ] https://github.com/metno/S-ENDA-Prototype
- [ ] a DOI of the code repo 
- [ ] ...

and the `<link to license>` is:
- [ ] https://raw.githubusercontent.com/metno/S-ENDA-Prototype/master/LICENSE
- [ ] https://github.com/metno/S-ENDA-Prototype/blob/master/LICENSE
- [ ] LICENSE
- [ ] ...

Comment (mortenwh) [reply source/definition_of_done.rst]: Better now?

**Pull request merged**